en-US/Punctuation.xml

```diff
@@ -754,7 +754,9 @@ PLAY [Gather and display facts for managed nodes] ****************************</
 		</para>
 		<para>
             For all other characters, use the character name followed by the symbol in parentheses.  
-            When referring to a symbol in parentheses, the symbol name comes after any noun that describes the symbol.			
+            For special character names that include a noun such as "sign", "symbol", or "character", provide the character name, the symbol in parentheses, and then the noun that describes the symbol, for example "the greater than symbol (>)". 
```

Comment:
Based on the description, I think the example needs to be updated to: "the greater than (>) symbol"